packages/turf-destination/test.js

```diff
@@ -24,7 +24,7 @@ test('turf-destination', t => {
         const bearing = (properties.bearing !== undefined) ? properties.bearing : 180;
         const dist = (properties.dist !== undefined) ? properties.dist : 100;
 
-        const dest = truncate(destination(geojson, dist, bearing));
+        const dest = truncate(destination(geojson, dist, bearing, { ...properties }));
```

Comment:
I am not sure but the contribution guide states "Turf aims to use ES5 features where rational. We do not use ES6 features."  I think de-structuring is ES6?

Reply:
Quite right, I had missed that this feature wasn't in es5. I can fix, especially since we are really just interested in the units anyway.